### DISCUSSION TITLE
Flows not working with sendmail

### DISCUSSION THREAD

**Discussion by dieriba** [directus/directus#15923] [Archive]
Hello, As my post says flows on directus not working with sendmail, I created a flow where for each new user creation an email is sent to them. I installed sendmail because I had an error due to that but event though I installed it is still not working. And this times no logs or errors is shown to me.
Can someone help me understand what’s happening please.

**Comment by br41nslug** [comment]
Email is inherently unreliable and a pain to setup 😬 After installing `sendmail` how have you configured it? https://kenfavors.com/code/how-to-install-and-configure-sendmail-on-ubuntu/
It also is very possible that the mails are being sent but blocked upstream in the network so double checking if your IP hasnt been blacklisted never hurts https://mxtoolbox.com/blacklists.aspx . If you're convinced mails are sending but not arriving then you can use a tools like https://www.mail-tester.com/ to see potential misconfigurations that could end up in being blocked.

**Reply by dieriba** [reply]
The only configuration i did was in the /etc/hosts where I added localhost:8055. In the link you given me what they mean by hostname ? Is it the port where my server is ? http://localhost:8055 ? 
Do I need to setup hosts folder as 

127.0.0.1 localhost http://localhost:8055
Or
127.0.0.1 http://localhost:8055 localhost
Or
Sould i write my localhost withouth my http.

Given that when I created an user now, flows for sending mail is trigered but I receive nothing. I think that maybe they're a misconfiguration from my sendmail because directis log's Flow is empty
